libs/lb-fs/src/fs_impl.rs

```diff
@@ -56,7 +56,7 @@ impl NFSFileSystem for Drive {
         let entry = data.get_mut(&id).unwrap();
         let id = entry.file.id;
 
-        let mut doc = self.lb.read_document(id).await.unwrap();
+        let mut doc = self.lb.read_document(id, false).await.unwrap();
```

Comment:
would consider changing these to true down the line, it's just a very noisy protocol